UI/graphics/background1.svg

```diff
@@ -0,0 +1 @@
lass="cls-19"><g class="cls-20"><rect class="cls-21" x="472.1" y="362.39" width="13.35" height="151.03" transform="translate(481.29 -210.92) rotate(47.78)"/><rect class="cls-21" x="513.79" y="416.53" width="6.68" height="151.03" transform="translate(533.96 -221.57) rotate(47.78)"/><rect class="cls-21" x="481.65" y="381.52" width="28.98" height="151.03" transform="translate(501.16 -217.51) rotate(47.78)"/></g></g><path class="cls-17" d="M476.54,549.24l2.54-6.9s3.84,4.21-1.54,13.79C472.55,565,476.54,549.24,476.54,549.24Z"/><path class="cls-11" d="M453.1,551.91s11.79,13.58,22.49-1.23c.66,1.89,2,7.08,2,7.08s-11.11,18.82-27.72,3.29C446.4,557.85,453.1,551.91,453.1,551.91Z"/><path class="cls-9" d="M424,593.55c-1.15,4.84,1.9,78.89,2.93,78.89a18.83,18.83,0,0,1,5.84-.91c2.58.18,4.86-49.85,6.36-57.5,2.87,9.7,7.53,58.06,9.87,58.69,3,.79,5.37,2.32,5.63-1.53,1.39-20.07-1.48-78.89-6.18-83.28C443.28,583.12,425.75,586,424,593.55Z"/><path class="cls-22" d="M457.8,676.11c0,2.79-3.38,2.86-5.35,2.86s-4.05,0-4.05-4.23c0-2.79,1.48-5.42,3.45-5.42S457.8,673.31,457.8,676.11Z"/><path class="cls-22" d="M435.09,676.11c0,2.79-3.38,2.86-5.36,2.86s-4,0-4-4.23c0-2.79,1.48-5.42,3.45-5.42S435.09,673.31,435.09,676.11Z"/><path class="cls-5" d="M438.24,532.63s-1.63,10.61-4.87,13.43,9.74,2.13,9.74,2.13-1-5.37,1.72-5.58S438.24,532.63,438.24,532.63Z"/><path class="cls-11" d="M433.19,542.4c.41,4.05-5,4.93-7.67,6.13-5.34,2.38-4.41,41.61-1.57,45,0,0,21.25,4.91,26.09-.61,2.38-2.72,3.06-41,3.06-41-1.65-1-9.75-4.2-9.48-6.4C443.77,544.25,434.26,542,433.19,542.4Z"/><path class="cls-17" d="M448.78,532.61c0,4.24.65,10.6-1.55,11.37-1.94.68-5.24-1.82-6.5-4s-.56-4.27-.56-6.45c0-4.24-3.31-8.63.93-8.63A7.68,7.68,0,0,1,448.78,532.61Z"/><path class="cls-9" d="M432.62,534.9c-1.48-4.69.47-10.3,6.32-11.39,2.81-.52,4.56.31,7,1.92,1.64,1.08,3.49,4.25,5.79,3.17-1.88,3.72-4,.06-6.22,1.53-1.18.77-.66,4.13-2,4.52a14,14,0,0,1-2.41-.12c-1.2,2.38,1,6.43-3.27,5.48C435.72,539.53,433.26,536.89,432.62,534.9Z"/><path class="cls-11" d="M285.52,339.15h-62l-10,72.9-17.8-72.9A24.59,24.59,0,0,1,171,314.56V279.82a24.6,24.6,0,0,1,24.59-24.6h89.89a24.61,24.61,0,0,1,24.6,24.6v34.74A24.6,24.6,0,0,1,285.52,339.15Z"/><path class="cls-9" d="M259.55,281.71c0,10.49-8.48,30.44-19,30.44-3.1,0-6-1.73-8.58-4.45-6.18-6.5-10.39-18.6-10.39-26a19,19,0,1,1,38,0Z"/><path class="cls-10" d="M240.57,262.73v49.42s-3.6-1.66-8.58-4.45c-6.18-6.5-10.39-18.6-10.39-26A19,19,0,0,1,240.57,262.73Z"/><rect class="cls-4" x="231.81" y="307.89" width="17.53" height="17.53"/><path class="cls-4" d="M237.49,303.36c0,3.78-.78,6.85-1.75,6.85s-1.74-3.07-1.74-6.85.78-6.85,1.74-6.85S237.49,299.57,237.49,303.36Z"/><path class="cls-4" d="M247.15,303.36c0,3.78-.78,6.85-1.74,6.85s-1.75-3.07-1.75-6.85.78-6.85,1.75-6.85S247.15,299.57,247.15,303.36Z"/><line class="cls-23" x1="231.81" y1="313.83" x2="249.34" y2="313.83"/><line class="cls-23" x1="231.81" y1="319.22" x2="249.34" y2="319.22"/><rect class="cls-4" x="237.49" y="325.41" width="6.18" height="3.84"/></g></g></g></svg>
```

Comment:
would you mind adding an empty line at the end of each file??

Reply:
Ok. Let me add it.